src/ring_theory/ideal/operations.lean

```diff
@@ -216,6 +218,8 @@ begin
   rw [ring_hom.map_mul, hφ1, mul_one]
 end
 
+/-- The homomorphism from `(∏ i, R)/(∏ i, f i)` to `∏ i, (R / f i)` featured in the Chinese
+  Remainder Theorem. It is bijective if the ideals `f i` are comaximal. -/
```

Comment:
I don't think this docstring is quite right. The definition involves the greatest lower bound of ideals, not products.

Reply:
You're totally right, sorry.
I think I'd opt to use `\bigcap` instead of `\bigsqcap` in the docstring, as I'm pretty much writing math  there, but I'd be happy with it either way.

Reply:
Yes, sounds good!